frontend/src/store/useAuthStore.js

```
@@ -55,30 +68,98 @@
+      );
 
       if (response.status !== 200) {
-        throw new Error('Logout failed');
+        throw new Error("Logout failed");
       }
 
       set({ user: null, authError: null });
       console.log("Logout successful");
+      return { success: true };
     } catch (error) {
       set({ authError: "Error logging out: " + error.message });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)